com.github.pyfa_org.Pyfa.json

```diff
@@ -0,0 +1,347 @@
+{
+  "id" : "com.github.pyfa_org.Pyfa",
+  "branch" : "stable",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "43",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "pyfa",
+  "finish-args" : [
+    "--share=ipc",
+    "--share=network",
+    "--socket=x11",
+    "--filesystem=xdg-config/gtk-3.0"
+  ],
+  "modules" : [
+    {
+      "name" : "python-3.7",
```

Comment:
why do you need to build python? It is in the runtime.

Reply:
Pyfa's official releases are built with python-3.7. Application has unresolved issues when running on recent versions of Python.